### PR TITLE
Add E2E to make sure each admin page loads

### LINKED ISSUE #1235: E2E: Add spec to make sure each admin page loads
Add a flow to our e2e specs to test each admin page - deposits, transactions, disputes. To start we can make sure the page loads as expected.

Related: #1234

### PR DESCRIPTION
**Pull request by ismaeldcom**
Fixes #1235

#### Changes proposed in this Pull Request

Opted to move every functions from `utils/index.js` to a new file called `utils/helpers.js`, leaving index like a barrel file. To avoid requieres cycles importing `clearAndFillInput` in flows from index. Also, I decided to unify export modules in favor of named exports.

Implemented a spec for each admin page (deposits, transactions, disputes) to test that page loads. I choose to use `toMatchElement` instead of `toMatch` to avoid false positives checking common text like `Disputes` against the full page text.

I think this cover the initial scope of the issue, let me now if I missed something.

#### Testing instructions

You can test each new spec running:
`npm run test:e2e-dev tests/e2e/specs/admin/deposits.spec.js`
`npm run test:e2e-dev tests/e2e/specs/admin/transactions.spec.js`
`npm run test:e2e-dev tests/e2e/specs/admin/disputes.spec.js`
or
`npm run test:e2e-dev tests/e2e/specs/admin/*`

For a headless run use `npm run test:e2e ...`

-------------------

- [x] Added changelog entry (or does not apply)
- [x] Covered with tests (or have a good reason not to test in description ☝️)
- [x] Tested on mobile (or does not apply)

### CODE REVIEW

File: tests/e2e/specs/admin/disputes.spec.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+const WCPAY_DEPOSITS =
+	config.get( 'url' ) +
+	'wp-admin/admin.php?page=wc-admin&path=/payments/disputes';
+
+describe( 'Admin disputes', () => {
+	beforeAll( async () => {
+		await StoreOwnerFlow.login();
+	} );
+
+	it( 'loads disputes', async () => {
```
Comment (dmallory42) [inline tests/e2e/specs/admin/disputes.spec.js:20]: Perhaps we should tweak the wording of the `it()` to make them a bit more descriptive. Currently, it'll just print: `Spec name: Admin disputes loads disputes, description: loads disputes` when the test is run.

What do you think about making the `it` value something like `page should load without any errors.`?

File: tests/e2e/specs/admin/transactions.spec.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+const WCPAY_DEPOSITS =
+	config.get( 'url' ) +
+	'wp-admin/admin.php?page=wc-admin&path=/payments/transactions';
+
+describe( 'Admin transactions', () => {
+	beforeAll( async () => {
+		await StoreOwnerFlow.login();
+	} );
+
+	it( 'loads transactions', async () => {
```
Comment (dmallory42) [inline tests/e2e/specs/admin/transactions.spec.js:20]: Same as comment in disputes section re: wording. :)

File: tests/e2e/specs/admin/deposits.spec.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+const WCPAY_DEPOSITS =
+	config.get( 'url' ) +
+	'wp-admin/admin.php?page=wc-admin&path=/payments/deposits';
+
+describe( 'Admin deposits', () => {
+	beforeAll( async () => {
+		await StoreOwnerFlow.login();
+	} );
+
+	it( 'loads deposits', async () => {
```
Comment (dmallory42) [inline tests/e2e/specs/admin/deposits.spec.js:20]: Same as comment in disputes section re: wording. :)

**Review by dmallory42 — CHANGES_REQUESTED** [review]
This looks great and works well, I think the changes you made to move the functions to a `helpers.js` file is good and makes the code cleaner!

I left a few minor comments with suggestions, let me know what you think and then this should be good to go. 👍

File: tests/e2e/specs/admin/transactions.spec.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+ * External dependencies
+ */
+import config from 'config';
+
+/**
+ * Internal dependencies
+ */
+import { StoreOwnerFlow } from '../../utils';
+
+const WCPAY_DEPOSITS =
```
[the remark this answers is not in the document]
Comment (ismaeldcom) [reply tests/e2e/specs/admin/transactions.spec.js:11]: 😅 Forgot to rename afte cope&paste

File: tests/e2e/specs/admin/disputes.spec.js (hunk 1 above)
Comment (ismaeldcom) [reply tests/e2e/specs/admin/disputes.spec.js:20]: 💯 Agree, I need to think better tests descriptions next time.

**Review by dmallory42 — APPROVED** [review]
Looks good to me! I think this is ready to 🚢 !

**Pull request merged**